### ISSUE TITLE
Reviewer shortage

### ISSUE THREAD

**Issue by amyjko** [acm-toce/documentation#17]
There is a fundamental reviewing labor disconnect that is going to be increasingly difficult:

* Our community is small (but growing)
* Submissions are growing substantially
* Reviewer availability is decreasing
* There are no inherent incentives to review (e.g., it's not valued in tenure and promotion)
* The pandemic has led to substantial burnout

How will we sustain a volunteer community of reviewers that sustains the peer review process?

**Comment by amyjko** [comment]
I think the only way to fix this is to incentivize reviewing. Here's one sketch of a model for this:

* When authors submit, one or more authors must commit to joining the reviewer pool, and to reviewing a total of 3 papers (to account for the three reviews they created). These commitments can all be one person, or split across authors. Each committing author needs to provide a statement of their expertise (phenomena and methods). 
* Submissions will be reviewed independent of whether authors have completed reviewing commitments, to ensure timely reviewing. However, the authors in the pool must accept up to the number of review commitments for the paper before the submission recommendation is released. For example, if they committed to 3, and were invited to do 3, but declined 2, the decision wouldn't be released until they do one more. But if they committed to 3, and were not invited to do any, the decision would be released.
* When an AE invites someone to review from the pool, they are reminded of their remaining reviewing commitments, if they have any in the pool, and of the consequences of not reviewing.
* Expectations for people in service roles would be reduced in recognition of their outsized contributions to peer review: AE authors and SIGCSE conference program committee members would only need to commit to 1 review, EiCs and program chairs on SIGCSE conferences or the CSE journal 0. All other submissions would need to commit 3 across the submission's authors.

This model loosely ties submitting to reviewing, but doesn't strictly tie it, as the expertise needed might not align with the expertise available. It also incentivizes authors to aggressively accept reviewing requests, to not risk delaying a publication decision. It also has the side benefit of creating a list of potential reviewers over time from the union of all submitting authors, and fresh metadata about their expertise.

One potential downside is that it may create an easy path to decline a review invitation if someone has no obligation, by replacing the altruistic motive with a review currency motive. That may be a risk worth taking to make it sustainable.

**Comment by amyjko** [comment]
An alternative that gets around some of the problems above:

* When authors submit, they are added to the review pool with 3 reviewing commitments (spread across multiple authors if they wish). Review pool members must accept the number of invites equal to their commitments. Afterwards, they may decline. Commitments accumulate for multiple submissions.
* Papers are reviewed as normal.
* Papers with any authors that have not made any progress on their reviewing commitments by accepting reviewing invites and and submitting reviews in the past 6 months are not eligible to submit until they do. Their submissions will be unsubmitted.

This also creates a strong incentive to watch for and accept invites and to write reviews in a timely manner.

**Comment by amyjko** [comment]
The reviewer pool is working okay for this; we're getting some options and people are using them.

**Comment by amyjko** [comment]
Made a request to SIGCSE-members.

**Comment by amyjko** [comment]
I've been reflecting more on the reviewer shortage. Ultimately, I think we're limited by the structural issues: there is a strong incentive to create reviewing work and a weak incentive to do it. I don't think there's anything we can do to address this structural issue without changing the tenure and promotion guidelines globally to reduce expectations of research volume and increase expectations of service.

However, I do think there are some things we can do to slightly increase incentives in this system, and balance out work. Here's my brainstormed list of incentives, divided up as "carrots" (things reviews want) and "sticks" (moral and social obligations):

Carrots

- Learning: Reviewers get to see other reviewer and AE comments after decisions
- Learning: Reviewers get explicit reviewer guidelines
- Learning: Reviewers can get constructive feedback on their reviews from AE and EiC
- Status: Opportunity to join the Editorial Board in the future
- Wellbeing: Journal reviewing has a more human pace and volume than conference reviewing
- Impact: Shape the discourse
- Community: Join Discord to discuss reviewing, research, guidelines
- Recognition: We can offer a public recognition of all reviewers who contributed their expertise
- Recognition: We could (liberally) offer "Excellent Review" distinctions, nominated by AE, approved by EiC, for inclusion on CVs.

Sticks

- Obligation: Submitting means having your name added to the reviewing pool, which increases the likelihood of an AE asking you to review, and an expectation that you will say yes
- Obligation: A templated message to authors after submission reminding them of their reviewing contributions to the journal over the last 12 months, and an expectation that they do as much work as they create, when asked.

For all of these, we would update the website, reviewing invitations, and submission confirmations to make these benefits and obligations clear.

Feedback from the board is greatly appreciated!

**Comment by bennytheshap** [comment]
All of the carrots and sticks seem reasonable to me. Great list.

**Comment by barbarer** [comment]
I like the stick ideas.  Some of the carrot ideas put more work on the AE.

**Comment by jbbAU** [comment]
Thanks, Amy, for always being proactive.  When reflecting a bit about why I am an AE, it mainly concerns "the care for the community" - it is needed for the "system" to function that we all take on this responsibility. So perhaps add a carrot: "Ensuring the well-functioning of the academic quality system" (someone can probably phrase it better)

**Comment by amyjko** [comment]
From a board member: 

> I vote for not liberally offering "excellent reviewer" but for actually making this a distinction that counts for something, perhaps recognizing both quality of reviews and overall quantity of service to the journal. I'm thinking about my department context where a department award (say, for teaching or mentoring) counts for very little but a college level award or a university level award counts for much more. We could document how someone earns "excellent reviewer" and make it clear that this actually counts for something at ACM TOCE.

**Comment by amyjko** [comment]
We continue to have chronic difficulties finding more than 2 reviewers per paper and many reviews are months late.

**Comment by amyjko** [comment]
I shared my idea with the board:

https://medium.com/bits-and-behavior/spring-sabbatical-update-hibernation-in-war-ae7ecaf43306

Here are a few of the sentiments I've received from the board so far:

* Let's try it! (The most common)
* There are lots of details to get right.
* I'm worried about external incentives displacing internal ones
* Don't address capacity, address process and criteria — peer review is too random.

**Comment by amyjko** [comment]
Drafted a proposal, revised based on board feedback, and sent out for public comment:

> Hello all,

> TLDR: Peer review in our community is increasingly unsustainable; ACM TOCE would like to address it, but needs your feedback before we proceed.

> For the past several years, we&#x27;ve been amidst a reviewer crisis in computing education research: the medium number of reviewer invites to yield three reviews for a journal submission has gone from 5 in 2015-2019 to about 9 in 2020-2023, with our average time to decision slipping from 3 months to 6. And the trends only appear to be getting worse, as more people are opting out of peer review, but continuing to submit papers for review. We are not alone — this has been happening&nbsp;[across academia](https://www.nature.com/articles/d41586-023-00403-8)&nbsp;for the past decade. This has generally only been visible to those of us in editor and program chair roles, tasked with finding the volunteer labor to review everyone&#x27;s work, but it&#x27;s a problem that we face collectively. It&#x27;s time to act if we wish to sustain our scholarly community.

> The ACM TOCE Editorial Board and I have a proposal and we&#x27;d like your feedback on it:

> https://docs.google.com/document/d/1RHirbCdQFxBeCbjAAbba1MJtxDOG4cuml66_xWGgXAI/edit#

> The short version is that we would like to link the ability to submit a paper to reviewing a certain amount, creating a token system that balances labor generated with labor done. We&#x27;re hopeful this system comes with many advantages, including sustainability, and that in the long run, more than ACM TOCE (and more than computing education) will switch to models like these.&nbsp;

> There are lots of subtle things to get right about this; read the proposal at the link above for the detailed proposal. You&#x27;re welcome to comment publicly on that document, or write me privately with feedback. And if you&#x27;re in an editing role and interested in participating in a pilot of this, please write me. All I ask is that if you do share feedback, you read the proposal in its entirety first — there are many questions you are likely to have, but the document likely addresses most of them, since its already been seen by many.&nbsp;Feedback by June 9th&nbsp;would be ideal.

>  Thank you!

**Comment by amyjko** [comment]
Poll results were conclusive: 80% of the community thinks we should pilot and most dissent is amongst full professors. Amy is assembling a team to build the necessary infrastructure.

**Comment by amyjko** [comment]
I've recruited a team to build the infrastructure necessary for this effort. We're kicking off the work this coming Monday.

**Comment by amyjko** [comment]
Got clarity on the board rationale for declining to participate in the pilot: they wanted TOCE to try it first and did not want to participate in any way until we had. (It's also clear that the board falsely views TOCE as somehow a separate community from SIGCSE, when in fact, we're all in the same community).

**Comment by amyjko** [comment]
All work related to this issue is for now happening at Reciprocal Reviews. Once we have viable infrastructure for a pilot, we'll plan the TOCE side of it here.

https://reciprocal.reviews/
